src/main/java/com/powsybl/balances_adjustment/pevf/PevfExchanges.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.balances_adjustment.pevf;
+
+import com.powsybl.timeseries.StoredDoubleTimeSeries;
+
+import java.time.ZonedDateTime;
+import java.util.*;
+
+/**
+ * Pan European Verification Function data.
+ *
+ * @author Thomas Adam {@literal <tadam at silicom.fr>}
+ */
+public class PevfExchanges {
+
+    /** Document identification. */
+    private String mRID;
+    /** Version of the document. */
+    private int revisionNumber;
+    /** The coded type of a document. The document type describes the principal characteristic of the document. */
+    private StandardMessageType type;
+    /** The identification of the nature of process that the
+     document addresses. */
+    private StandardProcessType processType;
+    /** The identification of the sender */
+    private String senderId;
+    private StandardCodingSchemeType senderCodingScheme;
+    /** The identification of the role played by a market player. */
+    private StandardRoleType senderMarketRole;
+    /** The identification of a party in the energy market. */
+    private String receiverId;
+    private StandardCodingSchemeType receiverCodingScheme;
+    /** The identification of the role played by the a market player. */
+    private StandardRoleType receiverMarketRole;
+    /** The date and time of the creation of the document. */
+    private ZonedDateTime creationDate;
+    /** This information provides the start and end date and time of the period covered by the document. */
+    private ZonedDateTime periodStart;
+    private ZonedDateTime periodEnd;
+
+    // Optional data
+    /** The identification of an individually predefined dataset in a
+     data base system (e. g. Verification Platform). */
+    private String datasetMarketDocumentMRId;
+    /** The identification of the condition or position of the document with regard to its standing. A document may be intermediate or final. */
+    private StandardStatusType docStatus;
+
+    // Time Series
+    private final Map<String, StoredDoubleTimeSeries> timeSeriesById = new HashMap<>();
+
+    PevfExchanges() {
+    }
+
+    // MarketDocument metadata
+    public String getMRId() {
+        return mRID;
+    }
+
+    public PevfExchanges setMRId(String mRID) {
```

Comment:
As a first approach we will consider that this class is immutable: make all the setters private package